oak_containers_hello_world_untrusted_app/src/main.rs

```diff
@@ -0,0 +1,29 @@
+//
+// Copyright 2023 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+mod service;
+use clap::Parser;
+
+#[tokio::main]
+async fn main() -> Result<(), anyhow::Error> {
+    let args = oak_containers_launcher::Args::parse();
+
+    tokio::select! {
+        _ = service::create(2, 6969) => {}
```

Comment:
We probably would want to have more sequencing here in the future, that is, don't start the launcher until the server is set up properly and listening. That'd avoid the possible race condition of the VM being up and running before the server is listening.

But this is fine for now.

Reply:
good point yes. In this example the service is up essentially instantly but you're correct that may not always be the case.